test/source/mock/fes/fes-constants.ts

```diff
@@ -142,6 +143,16 @@ export const processMessageFromUser3 = async (body: string, fesUrl: string) => {
   });
   expect(decrypted.success).to.equal(true);
   const decryptedMimeMsg = decrypted.content?.toUtfStr();
+  const cryptupDataRegex = /cryptup-data="([^"]*)"/;
+  const cryptupDataMatch = decryptedMimeMsg?.match(cryptupDataRegex);
+  expect(cryptupDataMatch).to.not.be.undefined;
+  expect(cryptupDataMatch && cryptupDataMatch.length).to.be.greaterThan(1);
```

Comment:
let's simplify `cryptupDataMatch && cryptupDataMatch.length` to `cryptupDataMatch?.length`

Reply:
will simplify the code. thank you.